dispatch/httpdispatch.go

```diff
@@ -47,42 +56,19 @@ type SenderConfig struct {
 	DeliveryInterval      time.Duration
 }
 
-type Dispatcher struct {
-	DispatchQueue    atomic.Value
+type HttpDispatcher struct {
```

Comment:
```suggestion
type HTTPDispatcher struct {
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms
HTTP should either be all lowercase or all uppercase.